cclib/bridge/cclib2ase.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Bridge for using cclib data in ASE (https://wiki.fysik.dtu.dk/ase/)."""
+
+import numpy as np
+
+from cclib.parser.data import ccData
+from cclib.parser.utils import find_package
+
+_found_ase = find_package("ase")
+if _found_ase:
+    from ase import Atoms, units
+    from ase.io.trajectory import Trajectory
+    from ase.calculators.calculator import PropertyNotImplementedError
+
+
+def _check_ase(found_ase):
+    if not found_ase:
+        raise ImportError("You must install `ase` to use this function")
+
+
+def makease(
+    atomcoords, atomnos, atomcharges=None, atomspins=None, atommasses=None
+):
+    """Create an ASE Atoms object from cclib attributes.
+
+    ASE requires atomic partial charges and atomic spin densities rather than
+    molecular charge and multiplicity, so we follow how other interfaces have
+    done (e.g., MOPAC, Gaussian and XTB) and require atomcharges and atomspins,
+    or leave undefined.
+
+    Inputs:
+        atomcoords - two dimensional array-like with atomic coordinates.
+        atomnos - one dimensional array-like with atomic numbers.
+        atomcharges - one dimensional array-like with atomic charges.
+        atomspins - one dimensional array-like with atomic spin densities.
+        atommasses - one dimensional array-like with atomic masses.
+    """
+    _check_ase(_found_ase)
+    return Atoms(
+        positions=atomcoords,
+        numbers=atomnos,
+        masses=atommasses,
+        charges=atomcharges,
+        magmoms=atomspins,
+    )
+
+
+def writetraj(filename, ccdata, popname="mulliken", index=None):

```

Comment:
Perhaps call this `write_trajectory`? I don't think the abbreviation is valuable here.

Reply:
Sure, done!